### DISCUSSION TITLE
Server

### DISCUSSION THREAD

**Discussion by arielsil** [microsoft/sudo#70] [Q&A]
Hi,

I am on 26212 Server Insider.  When might Server get SuDo ?

**Comment by joadoumie** [comment]
I wouldn't rule it out, but we don't currently have plans to light this up in server at the moment. We're still working on open-sourcing the project, and when we do, you'll be able to grab the latest releases from the GitHub repo and they should _just work_.

**Reply by joadoumie** [reply]
Following up here since we just added the source-code to the repository. We're working on adding Releases to the GitHub repo + making sudo is installable via winget which should help you get set up even if sudo does not ship out of the box for server.

**Reply by arielsil** [reply]
Great and thank you!  How do I winget it?

**Reply by roke-julian-lockhart** [reply]
@arielsil, see https://github.com/microsoft/sudo/issues/50#issue-2130028382.
